test/unit/ajax.js

```diff
@@ -1410,6 +1410,22 @@ QUnit.module( "ajax", {
 		};
 	} );
 
+	ajaxTest( "jQuery.ajax() - don't escape %20 in contentType (gh-4119)", 1, function( assert ) {
```

Comment:
We weren't escaping `%20` in contentType but in data. How about:
```suggestion
	ajaxTest( "jQuery.ajax() - '%20' in data with a header Content-Type override (gh-4119)", 1, function( assert ) {
```
? A bit long but I don't have shorter ideas at the moment.

Reply:
I've added my suggestion, based on yours.